harness/scoring/judge_anchored.py

```
@@ -88,38 +94,110 @@ def judge_answer(
+        m = re.search(r"try again in\s*([0-9]+\.?[0-9]*)\s*(ms|s)", msg, flags=re.I)
         if m:
             try:
-                return float(m.group(1))
+                val = float(m.group(1))
+                return val / 1000.0 if m.group(2).lower() == "ms" else val
             except Exception:
-                return 0.0
+                pass
         return 0.0
+    
+    def _extract_text(resp_obj: Any, source: str) -> str:
```

Ruff (0.14.3)

Dynamically typed expressions (typing.Any) are disallowed in `resp_obj`

(ANN401)